update

sambacha opened the pull request.

Commit:
As announced [1], Ubuntu Eoan is EOL.

[1] https://lists.ubuntu.com/archives/ubuntu-announce/2020-July/000258.html

tianon committed:
Merge pull request #182 from powersj/eol-eoan

As announced [1], Ubuntu Eoan is EOL.

tianon committed:
Move down "apt-get indextargets" to avoid creating binary blobs

mwhudson committed:
Merge pull request #187 from tianon/apt-get-indextargets

Move down "apt-get indextargets" to avoid creating binary blobs

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/freight-trust/docker-core?pullRequest=2) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/freight-trust/docker-core?pullRequest=2) before we can accept your contribution.<br/>**0** out of **4** committers have signed the CLA.<br/><br/>:x: powersj<br/>:x: mwhudson<br/>:x: tianon<br/>:x: docker-library-bot<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/freight-trust/docker-core?pullRequest=2) it.</sub>

Commit:
Add Ubuntu 20.10 Groovy as the rolling release

In preparation for Ubuntu 20.10 Groovy release later today make groovy the rolling release.

latest -> lts (defined by "latest" file)
rolling -> most recent release (defined by "rolling" file)
devel- > development series (defined by querying http://archive.ubuntu.com/ubuntu/dists/devel/Release)

tianon committed:
Merge pull request #191 from philroche/features/groovy-rolling

Add Ubuntu 20.10 Groovy as rolling release

vicamo committed:
add Ubuntu Hirsute Hippo

Signed-off-by: You-Sheng Yang <vicamo@gmail.com>

tianon committed:
Merge pull request #194 from vicamo/for-upstream/add-ubuntu-hirsute-hippo

add Ubuntu Hirsute Hippo

alex committed:
rolling is now hirsute

tianon committed:
Merge pull request #205 from alex/patch-1

rolling is now hirsute

vicamo committed:
add Ubuntu Impish Indri

Signed-off-by: You-Sheng Yang <vicamo@gmail.com>

tianon committed:
Initial conversion to https://partner-images.canonical.com/oci

tianon committed:
Merge pull request #206 from vicamo/for-upstream/add-ubuntu-impish

add Ubuntu Impish Indri

toabctl committed:
Move bionic, focal, groovy and hirsute to OCI tarballs

This is a follow up on 26b6ae75425961 . There are now tarballs on
https://partner-images.canonical.com/oci/ also for bionic, focal,
groovy and hirsute so use theses new tarballs.

tianon committed:
Merge pull request #208 from toabctl/move-more-suites-to-oci-tarballs

Move bionic, focal, groovy and hirsute to OCI tarballs

tianon committed:
Adjust GNUPGHOME cleaning to be more defensive against GnuPG edge cases

tianon committed:
Switch from SKS to Ubuntu keyserver

See also https://github.com/docker-library/faq#openpgp--gnupg-keys-and-verification and https://github.com/tianon/pgp-happy-eyeballs

Commit:
arch: add riscv64 images

Ubuntu now generates riscv64 images for Ubuntu 20.04 LTS (Focal) and
newer releases.

tianon committed:
Merge pull request #212 from powersj/add-riscv64

arch: add riscv64 images

toabctl committed:
As announced [1], Ubuntu Groovy is EOL

[1]
https://lists.ubuntu.com/archives/ubuntu-announce/2021-July/000270.html

tianon committed:
Merge pull request #215 from toabctl/groovy-eol

As announced [1], Ubuntu Groovy is EOL

alex committed:
rolling is now impish

tianon committed:
Merge pull request #218 from alex/patch-1

rolling is now impish

toabctl committed:
Add Ubuntu Jammy Jellyfish (22.04)

tianon committed:
Merge pull request #220 from toabctl/add-jammy

Add Ubuntu Jammy Jellyfish (22.04)

tianon committed:
Add new GitHub Actions workflow for updating "dist" branches

This workflow is manually triggered by going to the "Actions" tab, clicking on the appropriate workflow in the sidebar, and choosing the option to "Run" it (against the default branch).

This allows it to not only be managed more effectively (directly in GitHub) but also makes it run significantly faster since all architectures can update simultaneously.

tianon committed:
Merge pull request #228 from tianon/github-actions-workflow

Add new GitHub Actions workflow for updating "dist" branches

toabctl committed:
Ubuntu 21.04 Hirsute EOL

As announced [1], Ubuntu Hirsute has reached the end of it's life.

[1]
https://lists.ubuntu.com/archives/ubuntu-security-announce/2022-January/006363.html

woky committed:
Merge pull request #229 from toabctl/hirsute-eol

Ubuntu 21.04 Hirsute EOL

mwhudson committed:
make CONTRIBUTING.md truthier

tianon committed:
Merge pull request #233 from mwhudson/update-docs

make CONTRIBUTING.md truthier

darkdragon-001 committed:
Release Ubuntu Jammy Jellyfish

toabctl committed:
Merge pull request #235 from darkdragon-001/release-jammy

Release Ubuntu Jammy Jellyfish

woky committed:
Add Ubuntu Kinetic Kudu (22.10)

toabctl committed:
Merge pull request #237 from woky/master

Add Ubuntu Kinetic Kudu (22.10)

tianon committed:
Remove EOL Impish (21.10)

https://lists.ubuntu.com/archives/ubuntu-announce/2022-July/000281.html

tianon committed:
Merge pull request #244 from self-five/eol-impish

Remove EOL Impish (21.10)

docker-library-bot committed:
Update to 20220801 for amd64 (amd64)

woky committed:
update.sh: Fix manifest filename

At some time around 2022-06-11 the filename of published .manifest has
changed to be similar to .tar.gz filename. Previously it was:

  ubuntu-kinetic-oci-arm64-root.tar.gz
  ubuntu-kinetic-oci-arm64.manifest

Currently it is

  ubuntu-kinetic-oci-arm64-root.tar.gz
  ubuntu-kinetic-oci-arm64-root.manifest

mwhudson committed:
Merge pull request #246 from woky/master

update.sh: Fix manifest filename
